### DISCUSSION TITLE
How to scroll through the console contents (previously executed commands and their output) using only the keyboard?

### DISCUSSION THREAD

**Discussion by claudio-salvio** [microsoft/terminal#12728] [Q&A] [answered]
Hello,

I am wondering if there is currently any way to **scroll through executed commands and previous output** using only the **keyboard**.
Obviously I mean what is still available, not what was removed using `Clear-Host` (or the alias `cls`).
For example: in the **PowerShell ISE console pane** this could be done using the key combination `CTRL+UP`.
Using that combination we would "exit the current line" of the console and go through the history displayed on the screen.
That way you could -for example- copy a piece of text from the output of a command without having to use the mouse.

Best regards,
Claudio

**Comment by zadjii-msft** [comment, marked as answer]
That's a feature we're working on right now!

More or less, the spec is in #5804. I'd go follow #715 for updates!

**Reply by claudio-salvio** [reply]
Hello @zadjii-msft 
Thank you very much for the information.
This is very good news.
Best regards,
Claudio Salvio
